src/pages/Landing.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useOidc } from '@axa-fr/react-oidc';
+import { TableCellsIcon, UserIcon } from '@heroicons/react/24/solid';
+import { getLogger } from '../logger';
+import { Panel, PanelBody } from '../components/Panel';
+import { Button } from '../components/Button';
+import './Landing.css';
+
+const logger = getLogger('<Landing>');
+
+const Landing = () => {
+  const { login, isAuthenticated } = useOidc();
+
+  return (
+    <Panel
+      fill={false}
+      className="landing-panel"
+    >
+      <PanelBody>
+        <h3 style={{ fontWeight: '600' }}>Welcome to the Philanthropy Data Commons pilot.</h3>
+        <p>
+          This data viewer displays the data in the Philanythropy Data Commons pilot.
+          {' '}
+          <a
+            href="https://philanthropydatacommons.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Read more about the project here.
+          </a>
+        </p>
+        {isAuthenticated ? (
+          <Link
+            to="/proposals"
+            className="button button--color-blue button--inverted"
+          >
+            <TableCellsIcon className="icon" />
+            Go to the Dashboard
+          </Link>
+        ) : (
+          <Button
+            color="blue"
+            inverted
+            onClick={() => { login('/').catch(logger.error); }}
+          >
+            <UserIcon className="icon" />
+            Log in
+          </Button>
+        )}
+        <dl>
+          {!isAuthenticated && (
+            <div className="dli">
+              <dt>Need an account?</dt>
+              <dd>
+                Email Jim McGowan:
+                {' '}
+                <a href="mailto:jimmcgowan@opentechstrategies.com?subject=PDC%20Pilot%20account">
```

Comment:
Same suggestion here regarding configuration / not hardcoding values like this.

Reply:
(Alternatively, as a middle ground solution it might feel less sore if we made it more generic than Jim's name and email address specifically, e.g. something more like "admin@...")